odoo/cli/server.py

```diff
@@ -60,6 +60,8 @@ def report_configuration():
     if os.path.isfile(config.rcfile):
         _logger.info("Using configuration file at " + config.rcfile)
     _logger.info('addons paths: %s', odoo.modules.module.ad_paths)
+    if config.get('migrations_path'):
```

Comment:
To make the option name future-proof and correct english, I suggest to name it `upgrades_path` from day one, including in stable. We could later improve consistency by supporting `upgrades` along with `migrations` in the other locations where we load the scripts. Might be worth splitting the PR in 2: one for 10.0 with the current (minimal) changes, and one for master with support for the new names in other locations.